composer.json

```diff
@@ -13,7 +13,7 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0"
+        "php": ">=5.4.0"

```

Comment:
I don't think is good making the Router require `5.4.0` php version on this point. It could break the installation of people that are using it. Does your changes really need php 5.4.0?